stylix/palette.nix

```diff
@@ -46,6 +24,7 @@ in {
     (lib.mkRemovedOptionModule [ "stylix" "palette" "base0D" ] "Using stylix.palette to override scheme is not supported anymore")
     (lib.mkRemovedOptionModule [ "stylix" "palette" "base0E" ] "Using stylix.palette to override scheme is not supported anymore")
     (lib.mkRemovedOptionModule [ "stylix" "palette" "base0F" ] "Using stylix.palette to override scheme is not supported anymore")
+    (lib.mkRemovedOptionModule [ "stylix" "image" ] "Using stylix.image to override scheme is not supported anymore, use the constructor based approach described on the website instead")
```

Comment:
I think it's possible to have this transfer the image to the new option, and print a warning, rather than an error. That would avoid breaking old configs.